serverless.yaml

```diff
@@ -153,6 +153,16 @@ functions:
           maximumRetryAttempts: 3
           startingPosition: LATEST
 
+  startCrawler:
+    timeout: 30
+    memorySize: 512

```

Comment:
Since the lamba function is just making an AWS Api call, can the memory size be reduced from 512?

Reply:
Changed to 192